.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
-sudo: false
+sudo: required
```

Comment:
I remember sudo is required to use xenial dist. Will this be always like this? can we add a comment or a link to an issue we can track to know when we can remove sudo?

Reply:
`sudo: required` forces it to use a non-containerized infrastructure. Travis is deprecating their containerized infrastructure in favor of VM based.

See https://stackoverflow.com/questions/26299552/travis-sudo-is-disabled